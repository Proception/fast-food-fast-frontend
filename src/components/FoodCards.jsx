import React, { Component } from 'react';

const FoodCards = ({ menu }) => (
    <ul className="list-group list-group-flush">
      {menu.map(item => (
        <div className="order-item" key={item.id}>
          <img src={item.imageUrl}/>
          <h5>{item.title}</h5>
          <p><label for="price">Price :</label><span>&#8358;{item.price}</span></p>
          <p><label for="quantity">Qty :</label><input type="number" name="quantityqqq" min="0" max="100" step="1" value="1"/></p>
          <button className="button" name="" value="Add" id={item.id}>Add</button>
        </div>
      ))}
    </ul>
  );
export default FoodCards;
