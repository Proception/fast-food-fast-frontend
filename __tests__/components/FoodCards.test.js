import React from 'react';
import { shallow } from 'enzyme';
import FoodCards from '../../src/components/FoodCards.jsx';
import props from '../../__mocks__/foodCardsMock';

let wrapper;
beforeEach(() => {
  const { item } = props;
  wrapper = shallow(<FoodCards item={item}/>);
});
describe('FoodCards Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
