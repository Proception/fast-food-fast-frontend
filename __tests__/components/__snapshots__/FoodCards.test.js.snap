// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodCards Component should match snapshot 1`] = `
<div
  className="order-item"
>
  <img />
  <h5 />
  <p>
    <label
      htmlFor="price"
    >
      Price :
    </label>
    <span>
      â‚¦
    </span>
  </p>
  <button
    className="button"
    name=""
    value="Add"
  >
    Add
  </button>
</div>
`;
